.idea/watcherTasks.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectTasksOptions" suppressed-tasks="Babel" />
+</project>
```

Comment:
Hmm... Did this file get in by mistake?

Reply:
I thought I had successfully removed that and ignored it on my end. I'll take care of it.

Reply:
@polytypic It seems I successfully removed the errant file in the next commit - [see here](https://github.com/calmm-js/partial.lenses/pull/121/files)?

Reply:
Sorry, I'm obviously off base on that previous comment 😄 

It may make more sense for me to put this aside for now and take a look again tomorrow.

Reply:
I believe I successfully took care of this via interactive rebase. It was a chore, as I ran into myriad issues with git and cygwin on my PC along the way. Those issues are still not resolved, but I soldiered my way through to a solution. Now to spend the rest of the day (probably) trying to get git and cygwin working correctly on this PC. 😦